code/modules/ghostroles/spawner/base.dm

```diff
@@ -116,10 +126,33 @@
 
 	return null //If we dont have anything return null
 
+//Selects a spawnatom from the list of available atoms and removes it if use is set to true (default)
+/datum/ghostspawner/proc/select_spawnatom(var/use=TRUE)
+	if(loc_type != GS_LOC_ATOM)
+		log_debug("Ghostspawner: select_spawnatom is not valid for spawner [short_name] as it is not atom pased")
+		return null
```

Comment:
```suggestion
		return
```

Reply:
null is quite intentional here, as I dont want to return . in case. is used some time later on.